doc/build-unix.md

```diff
@@ -194,7 +194,7 @@ Hardening Flags:
 
 Hardening enables the following features:
 
-* Position Independent Executable
+* _Position Independent Executable_  
```

Comment:
Remove trailing whitespaces.

Reply:
Well, that is the main change, double-space causes a linebreak: https://markdown-guide.readthedocs.io/en/latest/basics.html#line-return 

Is there a preferred way of doing linebreaks in this context?

Reply:
Either change formatting, update the failing `test/lint/lint-whitespace.sh` or leave as it is.

Reply:
Ah, didn't notice the failed check, sorry. I'll figure something out.